docs/requirements/apod/101-esx-layer1-lag1.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Documentation     aPod node LAG1 connectivity.
+...
+...               LAG1 currently consists of two physical interfaces connected to 2 upstream devices.
+...               Link aggregation is configured using LACP protocol.
+...               
+...               Link aggregation must be configured in LACP Active.
+...               Link aggregation must suspended individual ports.
+...               Link aggregation must use Fast LACP timer.
+...               
+...               aPod node LAG1 need connectivity to following networks:
```

Comment:
Who creates these networks in netbox? Who allocated the VLAN tag? Are these flat L2 networks or do you need a gateway in there? (Except CP mgmt where this question is answered)